### PR TITLE
Add the jetpack footer to the /log-in page

### LINKED ISSUE #14983: Login: Add missing `Powered by Jetpack` footer
I figured out we missed newly introduced `Powered by Jetpack` footer on new Login page. See how it looks on `/wp-login.php`:

image uploaded from ios

### PR DESCRIPTION
**Pull request by scruffian**
This pull request fixes #14983 by adding a "Powered by Jetpack" footer to the `/log-in` page.
  
screen shot 2017-06-12 at 12 33 52
  
#### Testing instructions
  
1. Run `git checkout add/log-in-footer` and start your server
2. Open the [`Log In` page](http://calypso.localhost:3000/log-in)
3. Check that the "Powered by Jetpack" footer shows at the bottom of the screen
  
#### Additional notes
Test in different window sizes, to make sure it doesn't overlap content.
  
#### Reviews
  
- [ ] Code
- [ ] Product

### CODE REVIEW

File: client/login/wp-login/style.scss [hunk 1]
```diff
@@ -1,4 +1,15 @@
-.wp-login.main {
+.is-section-login {
+	padding-bottom: 47px;
+	position: relative;
+	min-height: calc( 100vh - 47px );
+
+
+	.layout__content {
+		position: static;
```
Comment (scruffian) [inline client/login/wp-login/style.scss]: @mtias, what do you think about this technique for over-riding base styles?

**Comment by scruffian** [comment]
The tests are failing because the Jetpack logo has long lines. I think we should leave this file like this.

**Comment by breezyskies** [comment]
Comparing to the existing "Powered by Jetpack" on wp-login, it looks like this version gets hidden below the fold on many screen sizes (this actually might be more the result of the page content being shorter). Do we want to fix it to the bottom of the visible screen in all cases? I assume we're trying to make this prominent but the downside of course is that it would cover some of the links we have here, so either way there are tradeoffs.

**Comment by stephanethomas** [comment]
What's the reason for embedding this SVG into a React component instead of just linking to [the image](https://github.com/Automattic/wp-calypso/tree/master/public/images/jetpack)? The problem I see with this approach is that the browser can't really cache it (well, it's cached with the bundle but the bundle changes way more often than the image).

**Comment by gziolo** [comment]
> The tests are failing because the Jetpack logo has long lines. I think we should leave this file like this.

@scruffian can you use Eslint comment to disalbe this error in this file or move SVG to its own file as @stephanethomas suggested?

File: client/login/wp-login/style.scss [hunk 2]
```diff
@@ -24,3 +35,14 @@
 		border-bottom: none;
 	}
 }
+
+.wp-login__jetpack-footer {
+	background: #fff;
```
Comment (gziolo) [inline client/login/wp-login/style.scss]: It looks like this line has different spacing.

File: client/login/wp-login/style.scss [hunk 3]
```diff
@@ -1,4 +1,15 @@
-.wp-login.main {
+.is-section-login {
+	padding-bottom: 47px;
+	position: relative;
+	min-height: calc( 100vh - 47px );
+
```
Comment (gziolo) [inline client/login/wp-login/style.scss]: We can remove one empty line here.

**Comment by gziolo** [comment]
I noticed that footers have different height in Calypso and on WPCOM:
screen shot 2017-06-13 at 13 41 32
screen shot 2017-06-13 at 13 42 13

**Comment by scruffian** [comment]
> Comparing to the existing "Powered by Jetpack" on wp-login, it looks like this version gets hidden below the fold on many screen sizes (this actually might be more the result of the page content being shorter). Do we want to fix it to the bottom of the visible screen in all cases? I assume we're trying to make this prominent but the downside of course is that it would cover some of the links we have here, so either way there are tradeoffs.

Yes, the issue is that the log-in form is bigger in this version. I did play around with fixed position, but i felt like this option got a good balance between visibility of the banner, without hampering the utility of the links beneath the login form.

File: client/login/wp-login/style.scss (hunk 3 above)
Comment (scruffian) [inline client/login/wp-login/style.scss]: fixed

File: client/login/wp-login/style.scss (hunk 2 above)
Comment (scruffian) [inline client/login/wp-login/style.scss]: fixed

**Comment by scruffian** [comment]
> What's the reason for embedding this SVG into a React component instead of just linking to the image? The problem I see with this approach is that the browser can't really cache it (well, it's cached with the bundle but the bundle changes way more often than the image).

Done, thanks :)

File: client/login/wp-login/index.jsx [hunk 4]
```diff
@@ -96,30 +96,41 @@ export class Login extends React.Component {
 		return <LocaleSuggestions locale={ this.props.locale } path={ this.props.path } />;
 	}
 
+	renderFooter() {
+		return (
+			<div className="wp-login__jetpack-footer">
+				<img src="/calypso/images/jetpack/powered-by-jetpack.svg" style={ { width: '200px' } } />
```
Comment (drewblaisdell) [inline client/login/wp-login/index.jsx]: I'm also curious about why you chose inline CSS vs. updating the stylesheet.

Comment (gziolo) [inline client/login/wp-login/index.jsx]: Also why SVG has a different size: `<svg width="454px" height="38px" ...`?

Comment (scruffian) [inline client/login/wp-login/index.jsx]: This is how it's done in wp-login.php, but I agree it doesn't make much sense. I'll update.

**Comment by drewblaisdell** [comment]
LGTM :+1:

**Pull request merged**